src/components/EstimatedNetGrant.tsx

```diff
@@ -12,15 +12,20 @@ import {
   Heading,
 } from '@chakra-ui/react';
 import {
-  useCallback, useState, useMemo,
+  useCallback, useState, useMemo, ReactNode,
 } from 'react';
-import PropTypes from 'prop-types';
 import useForm from '../hooks/useForm';
 import yup from '../util/yup';
 import { getDollar, getPercentString } from '../util/formatters';
 import useAlphaVantageAPI from '../hooks/useAlphaVantageAPI';
 import RHFInput from './RHFInput';
 
+type CastedFormData = {
+  symbol: string | undefined,
+  stateTax: number | undefined,
+  ssEarnings: number | undefined,
+  sharesGranted: number | undefined,
+}
```

Comment:
A more idiomatic approach would probably be to make these optionals. 
```ts
symbol?: string,
```

Reply:
Ah, very good call.